spec/features/user_sees_own_reviews_spec.rb

```diff
@@ -6,23 +6,26 @@
   So that I can see all my reviews
 } do
 
-    # Acceptance Criteria
-    #
-    # [√] I want to see all of my reviews from my profile page
+  # Acceptance Criteria
+  #
+  # [√] I want to see all of my reviews from my profile page
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:author) { FactoryGirl.create(:author) }
+  let!(:genre) { FactoryGirl.create(:genre) }
+  let!(:book) { FactoryGirl.create(
+    :book,
+    user_id: user.id,
+    author_id: author.id,
+    genre_id: genre.id)
+  }
+  let!(:review) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.